src/configs/haskell.rs

```diff
@@ -0,0 +1,31 @@
+use crate::config::ModuleConfig;
+
+use serde::Serialize;
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig, Serialize)]
+pub struct HaskellConfig<'a> {
+    pub format: &'a str,
+    pub version_format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
+    pub disabled: bool,
+    pub detect_extensions: Vec<&'a str>,
+    pub detect_files: Vec<&'a str>,
+    pub detect_folders: Vec<&'a str>,
+}
+
+impl<'a> Default for HaskellConfig<'a> {
+    fn default() -> Self {
+        HaskellConfig {
+            format: "via [$symbol($version )]($style)",
+            version_format: "v${raw}",
+            symbol: " ",
```

Comment:
I can't actually see this symbol in the web browser due to font issues. Is this supposed to be a nerdfont symbol? If so, we'll need to change it over to unicode.

NerdFont symbols can go in the NerdFont preset if you'd like to preserve it.

Reply:
(Same applies to documentation for defaults).

Reply:
Yes, it is a NerdFont symbol. 
<img width="156" alt="image" src="https://user-images.githubusercontent.com/23358293/158562639-93c6681c-d80a-413d-bcfb-5612aed94889.png">

Moved to the NerdFont preset.